Makefile

```diff
@@ -35,6 +35,10 @@ test-e2e:
 test-e2e:
 	go test -timeout 30m ./e2e -test.v -ginkgo.v --binary "$(AWS_CONTAINERD_LOGGERS_BINARY)" --log-driver "awslogs"
 
+.PHONY: test-e2e-for-fluentd
+test-e2e-for-fluentd:
+	go test -timeout 30m ./e2e -test.v -ginkgo.v --binary "$(AWS_CONTAINERD_LOGGERS_BINARY)" --log-driver "fluentd"
```

Comment:
what happens when you run this without the server up and running?  anyway to get the fluentd server involved here, or possible in a second makefile command so people can test this locally a bit more easily?

Reply:
It will fail. It is a bit tedious to add more makefile targets now. I will polish the local test steps later in separated PRs, either by documentation or script. https://github.com/aws/shim-loggers-for-containerd/pull/66#discussion_r1327714177